src/native.js

```diff
@@ -174,7 +174,7 @@ function isOpenRTBAssetValid(asset) {
       logError(`for data asset 'type' property must be a number`);
       return false;
     }
-  } else if (asset.video) {
+  } else if (FEATURES.VIDEO && asset.video) {
```

Comment:
I'm not sure this is right; if you turn off video but not native, does that mean that you don't want video within native?

Reply:
Fair point. Reverted!